mediapicker/src/main/res/layout/oss_fragment_base.xml

```diff
@@ -1,17 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="vertical" android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
     <androidx.appcompat.widget.Toolbar
         android:id="@+id/baseToolbar"
         android:layout_width="match_parent"
-        android:background="@android:color/white"
         android:layout_height="wrap_content"
-        android:visibility="gone"/>
-    <include layout = "@layout/oss_custom_toolbar"/>
+        android:background="@android:color/white"
+        android:visibility="gone" />
+
+    <include
+        android:id="@+id/customToolbar"
```

Comment:
Is there any reason, why we added the id?

Reply:
for convenience to access the toolbarTitle, toolbarBackButton and etc here, if not, will need findViewId. that is also possible.

```
baseBinding.customToolbar.apply {
            toolbarTitle.text = getScreenTitle()
            toolbarTitle.setTextColor(requireContext().resources.getColor(R.color.oss_toolbar_text))
            if (setHomeAsUp()) {
                toolbarBackButton.visibility = View.VISIBLE
                toolbarBackButton.setImageResource(getHomeAsUpIcon())
            } else {
                toolbarBackButton.visibility = View.GONE
            }
            if (shouldHideToolBar()) {
                toolbarView.visibility = View.GONE
            }
            toolbarBackButton.setOnClickListener { onBackPressed() }
```